Mobile/ShopLens/ShopLensWeb/ShopLensContext.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.IO;
+using Microsoft.EntityFrameworkCore;
+
+namespace ShopLensWeb
+{
+    public class ShopLensContext : DbContext
+    {
+        private bool UseLocalDb { get; }
+        private string ConnectionString { get; set; }
+        
+        public ShopLensContext(string connectionString, bool useLocalDb = false)
```

Comment:
I don't think the false if not specified requirement is necessary, I think it just makes it really weird. You just pass the value from the App.Config but you have to pass it to the constructor, that's it. You don't need to think which parameters should I omit to use my Azure db. Just one constructor, second parameter always has to be passed.

Reply:
This is subjective, but right now the false if not specified requirement just causes confusion and the README really suffered in clarity because of it.

Reply:
Not sure I follow you on the reasoning, but I've removed the optional parameter :put_litter_in_its_place: